essential-cardano-list.md

```diff
@@ -467,47 +467,3 @@ Tools to help you build on Cardano:
 
 ### Other IOHK Solutions ###
 - [Mantis](https://mantisclient.io/)
-
-### 🥩 Stake Pools 🥩 ###

```

Comment:
Maybe we could replace this section with a short "finding a stake pool" section?

Reply:
We have a [exploring pools](https://github.com/input-output-hk/essential-cardano/blob/main/essential-cardano-list.md#exploring-stake-pools) section already on this page linking to various ways to find stake pool from different websites, but yes - happy for that section to be magnified instead